tests/test_diffraction_objects.py

```diff
@@ -406,7 +411,7 @@ def test_dump(tmp_path, mocker):
 @pytest.mark.parametrize(
     "do_init_args, expected_do_dict, divide_by_zero_warning_expected",
     [
-        (  # instantiate just array attributes
+        (  # Instantiate just array attributes
```

Comment:
Quick PEP8 standard, starting each comment with a cap letter

Reply:
Yes, let's use that convention.  For the test cases I don't mind if they are full sentences or not tbh as long as the intent is clear.